### PR TITLE
Support Ruby 3

### PR DESCRIPTION
**Pull request by almathew**
Doing the minimum work required right now to avoid rampant dependency mismatches downstream.

I'm sure someday we'll regret my choice not to pin activesupport and nokogiri, but right now it's super convenient to let bundle resolve the correct version instead.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'activesupport', '~> 4.0'
-gem 'nokogiri', '~> 1.9.1'
+gem 'activesupport'
```
Comment (fancyremarker) [inline Gemfile:3]: Instead, we should just remove these dependencies if we no longer care about a version. (They're being pulled in implicitly by Rails.)

**Review by fancyremarker — CHANGES_REQUESTED** [review]

**Comment by fancyremarker** [comment]
@almathew : (same comment as aptible/aptible_api_shared#47) as discussed in Slack, I'm leaving this open until we can confirm deploy-api tests pass on the updated gem; perhaps open a deploy-api PR with gems pinned to branches and assign me to review it?

**Comment by almathew** [comment]
I shared this in the aptible_api_shared repo, but downstream tests have been updated to use this version of fridge and are passing.

**Review by mdelaossa — APPROVED** [review]
since downstream is passing, approving

**Comment by almathew** [comment]
I'm merging with @mdelaossa 's approval because all of @fancyremarker 's requested changes were met.

**Pull request merged**